src/observer/ingest_logic.py

```diff
@@ -356,7 +359,67 @@ def regenerate_all():
     regenerate_many(logic.known_articles())
 
 #
-# upsert from file/dir of article-json
+# upsert article-json from api
+#
+
+if settings.ENV not in ['prod', 'continuumtest']:
```

Comment:
different in `end2end` vs `prod` is a smell to me, it should be included in the list?